src/exts/core/listener.py

```diff
@@ -63,6 +63,15 @@ async def on_message(self, message: Message) -> None:
 
             kwargs["embeds"].append(embed)
 
+        if message.stickers and message.stickers[0].format != StickerFormatType.lottie:
+            sticker = message.stickers[0]
+
+            embed = Embed(title=f"Sticker: `{sticker.name}`")
```

Comment:
I think the embed should use the `0x87CEEB` colour since it uses that colour for other embeds.

Reply:
Oh, good point, though the attachment embeds are also not using a color (which is what I based this code on :^) ):
https://github.com/vcokltfre/crosschat/blob/dc92272876db3306d85fcf5bf2e238477d07f70e/src/exts/core/listener.py#L59-L60

Reply:
Ahh, my bad, I see now, I'd say that that embed should also use that colour, but I suppose vco would have to decide :^)